src/crimson/os/seastore/seastore.cc

```diff
@@ -1591,10 +1591,10 @@ SeaStore::Shard::_clone_omaps(
   OnodeRef &d_onode,
   const omap_type_t otype)
 {
-  return trans_intr::repeat([&ctx, onode, d_onode, this, otype] {
+  return trans_intr::repeat([&ctx, onode, d_onode, this, otype]() mutable {
     return seastar::do_with(
       std::optional<std::string>(std::nullopt),
-      [&ctx, onode, d_onode, this, otype](auto &start) {
+      [&ctx, onode, d_onode, this, otype](auto &start) mutable {
```

Comment:
Probably it's better to adjust the type of the param rather than adding `mutable` here? @xxhdx1985126 

Still approving to fix the issue first.

Reply:
Yeah, please take a look: https://github.com/ceph/ceph/pull/55855

Reply:
Also approved.